android/sdk/src/main/java/org/jitsi/meet/sdk/connection_service/RNConnectionService.java

```diff
@@ -0,0 +1,128 @@
+package org.jitsi.meet.sdk.connection_service;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.RequiresApi;
+import android.telecom.DisconnectCause;
+import android.telecom.PhoneAccount;
+import android.telecom.PhoneAccountHandle;
+import android.telecom.TelecomManager;
+
+import com.facebook.react.bridge.LifecycleEventListener;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+/**
+ * The react-native side of Jitsi Meet's {@link ConnectionService}. Exposes the Java Script API.
+ *
+ * @author Pawel Domas
+ */
+@RequiresApi(api = Build.VERSION_CODES.O)
+public class RNConnectionService
+        extends ReactContextBaseJavaModule
+        implements LifecycleEventListener {
```

Comment:
Why are we implementing this listener, if the implemented methods are empty?

Reply:
yeah, good point, it was not empty at some point and maybe I copy pasted from another module